config.go

```diff
@@ -16,6 +16,8 @@ type ConfigType struct {
 	KeyFile     string            `yaml:"key_file"`
 	Admins      map[string]string `yaml:"admins"`
 	Censor      bool              `yaml:"censor,omitempty"`
+	Private     bool              `yaml:"private,omitempty"`
+	WhiteList   map[string]string `yaml:"whitelist"`
```

Comment:
This can be `omitempty`

Reply:
Indeed, it is better that way. Nice catch.